### DISCUSSION TITLE
OpenGPT-X

### DISCUSSION THREAD

**Discussion by bilo1967** [ggml-org/whisper.cpp#2607] [General]
Hi, 
The OpenGPT-X research project [has released](https://www.iais.fraunhofer.de/en/press/press-release-241126.html) as open source a large language model, available for download on Hugging Face as "Teuken-7B" ([info here](https://huggingface.co/openGPT-X/Teuken-7B-instruct-research-v0.4)), trained from scratch in all 24 official languages of the European Union.
Is there any possibility that it can be used with whisper CPP?

**Comment by MathiasSchindler** [comment]
At the risk of causing confusion, I would like to ask if this issue was really meant for whisper.cpp instead of llama.cpp (see https://github.com/ggerganov/llama.cpp/discussions/10539 for a discussion there)

**Reply by bilo1967** [reply]
Yes, sorry, mistakenly posted here.
Better to have a coffee before posting.
Thank you for the link at the right discussion.
